src/vorta/views/repo_add_dialog.py

```diff
@@ -59,6 +79,27 @@ def receive():
         dialog = choose_file_dialog(self, self.tr("Choose Location of Borg Repository"))
         dialog.open(receive)
 
+    def set_password(self, URL):
```

Comment:
This is nice. So it will pre-fill saved passwords?

Reply:
Yes, but only if the current entered password is empty.